src/compiler/compile.ts

```diff
@@ -33,9 +33,6 @@ export function compile(
     // Make sure to keep the original spec as-is
     const specCopy = JSON.parse(JSON.stringify(spec));
 
-    // Override default visual encoding (i.e., `DataTrack` => `BasicSingleTrack`)
-    overrideDataTemplates(specCopy);
-
```

Comment:
This function adds default visual encodings to DataTrack. Since there is no more DataTrack, we can remove this function